manpages/fcpgenkey.1

```diff
@@ -11,9 +11,28 @@ fcpgenkey \- generate a single Freenet SSK@/USK@ keypair
 fcpgenkey is a simple command\-line FCP client program for generating
 new Freenet SSK@ or USK@ public/private keypairs.
 
-The keypair generated is written to standard output stream, with
-the public key, then a newline, then the private key.
+The keypair generated is written to standard output stream as SSK@
+keys, with the public key, then a newline, then the private key.
 
+.SH "Converting an SSK@ key to an USK@ one"
+.LP
+USK@ are really just a user-friendly wrapper around SSK@, which hide
+the process of searching for more recent versions of a site. Any
+address written using an USK@ key can also be written using an SSK@
+key.
+.TP
+.B
+Syntax of SSK@ key:
+SSK@<key>/<name_of_site>-<version#>/
```

Comment:
How about the USK goes first and then the SSK is "Syntax of equivalent SSK@ key:"? SSKs don't have to be formatted that way.